Basic/ClientDriven/Assets/StarterAssets/ThirdPersonController/Scripts/ThirdPersonController.cs

```diff
@@ -96,7 +96,13 @@ public class ThirdPersonController : MonoBehaviour
         private int _animIDGrounded;
         private int _animIDJump;
         private int _animIDFreeFall;
-        private int _animIDMotionSpeed;
+        // MTT CHANGE START
+        // Reason: the Third Person Controller Animator uses a speed multiplier on the player's walk/run blend tree 
+        // state, to apply speed based on a player's input. On ghost clients, this component is disabled on ghost
+        // clients, meaning this parameter is at it's default value, and the walk/run blend tree will appear stuck.  
+        // To circumvent that, the walk/run blend tree state is not multiplied by any game-side code.
```

Comment:
mmm so wait, doesn't that reduce the amount of functionality from the original Starter Assets? If users want to change the character's speed, with this change here the animations wouldn't react to it? Why aren't we trying to sync that magnitude info? Or deduce it from the character's current speed?

Reply:
Followed up offline and this has since been re-enabled!